packages/filesystem/src/common/filesystem.ts

```diff
@@ -187,9 +187,6 @@ export interface FileSystem {
     getFsPath(uri: string): Promise<string | undefined>
 }
 
-/**
- * @deprecated since 1.4.0 - in order to support VS Code FS API (https://github.com/eclipse-theia/theia/pull/7908), use `FileService.access` instead
- */
 export namespace FileAccess {
```

Comment:
@westbury there is currently only one usage of `FileAccess.Contants` left in the codebase.
Should we remove it as part of the pull-request?

https://github.com/eclipse-theia/theia/blob/3420ef639e3b9240edd94d207d21d83f98a4e697/packages/filesystem/src/electron-browser/file-dialog/electron-file-dialog-service.ts#L85

We can then think of removing the deprecation in a future release (ex: `1.13.0` or possibly remove it now).

Reply:
I'm confused by this comment.  This PR is not removing the deprecated constants.  It is about removing the `@deprecated` because I believe the constants were incorrectly deprecated.  The constants should remain, indefinitely.  So this line is, I believe, correct.

The constants were, presumably, deprecated because they were used by FileSystem, and FileSystem was deprecated.  However, as this line demonstrates, the constants are also used by FileService.  So the constants should not be deprecated.

Reply:
Sorry I had misunderstood. I thought the constants were not useful anymore since we can simply use those provided by `node` itself:

https://github.com/eclipse-theia/theia/blob/3420ef639e3b9240edd94d207d21d83f98a4e697/packages/filesystem/src/node/download/file-download-handler.ts#L59

I'm not sure why there was confusion initially when `access` and `mode` should follow the same documentation available from `node` itself: (https://nodejs.org/api/fs.html#fs_fspromises_access_path_mode).

<img width="1425" alt="Screen Shot 2021-03-29 at 6 45 08 PM" src="https://user-images.githubusercontent.com/40359487/112909174-e7ad1c80-90be-11eb-883e-bbeeeaf47b40.png">

I assume you want to keep the constants instead of deprecating completely since you feel it is useful for callers to use (and we cannot use the `fs.constants` outside of `node`)?